golem_messages/message/base.py

```diff
@@ -353,13 +397,55 @@ class Hello(Message):
         'challenge',
         'difficulty',
         'metadata',
+        '_version',

```

Comment:
Why not pass it along? As far as I can tell, slots starting with `_` are not initialized.